packages/web/lib/hooks.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
can we remove this and set correct types?

Reply:
Added ESLint rule instead of ignoring in file. I think for hooks its fine to not annotate the return type since its usually based on things that are well typed / external to our code (e.g. useContext return type handled by React, useGetMeQuery return type handled by graphql-codegen).